llvm/lib/FuzzMutate/FuzzerCLI.cpp

```diff
@@ -84,9 +84,10 @@ void llvm::handleExecNameEncodedOptimizerOpts(StringRef ExecName) {
       Args.push_back("-passes=simplifycfg");
     } else if (Opt == "gvn") {
       Args.push_back("-passes=gvn");
+    } else if (Opt == "newgvn") {
+      Args.push_back("-passes=newgvn");
```

Comment:
Probably remove this one as well. It's also a non-production pass that has enough known issues that more reports are unlikely to be useful to anyone.

Reply:
Done